lib/OpenQA/Worker/WebUIConnection.pm

```diff
@@ -261,17 +261,6 @@ sub finish_websocket_connection {
     }
 }
 
-sub disable {
```

Comment:
Right, the only way a web UI connection is disabled is now when the registration fails so this is not required anymore. However, at some point I'd like to use it within `sub _handle_command_incompatible` (when refactoring I didn't want to change the behavior of that because I already had too many changes). So can you make the removal a separate commit so I can restore it individually if required?